### DISCUSSION TITLE
When cockpit is updated to backports?

### DISCUSSION THREAD

**Discussion by Ludo-code** [cockpit-project/cockpit#18557] [Q&A] [answered]
Hello everyone, i have wanted to know when cockpit is released to the debian/ubuntu backports? i have checked out and on fedora package for exemple is more updated than the debian/ubuntu version...

**Comment by martinpitt** [comment, marked as answer]
[Debian](https://tracker.debian.org/pkg/cockpit) has 286 in testing and backports, that's up to date. Debian is in rather deep freeze now, so I can't upload new  versions easily -- it now takes 20 days to migrate, and we release every 14 days. When all goes well, 287 will enter testing in 5 days, then I'll update backports.

[Ubuntu](https://launchpad.net/ubuntu/+source/cockpit) backports have 286, while devel has 287. I'll update it now.
